### ISSUE TITLE
Staging Review finding: Heading structure doesn't match content for POA request page

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#104055]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Accredited Rep Facing (ARF) + Accredited Rep Mgmt (ARM)
**Product:** ARF (Accredited Rep Portal MVP) + ARM (Appoint a Rep 2.0)
**Feature:** Form 21-22: Digital Submission (ARM) and Accept/Decline (ARF)

## Findings details
**VA.gov Experience Standard - issue:** [User can determine next steps in a flow.](https://design.va.gov/about/experience-standards/#user-can-determine-next-steps-in-a-flow)
**VA.gov Experience Standard - category:** [Findability](https://design.va.gov/about/experience-standards/#category-findability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

When viewing an individual POA request, the heading structure doesn't match the content. Screen reader users often use the heading hierarchy to help understand the page and navigate through it quickly, so when the heading structure doesn't match the content users can get lost.

Currently, the heading structure for the page goes:
- H1: POA request
   - H2: [Claimant name]
   - H2: Claimant information
   - H2: Authorization information

This implies that the claimant information, for instance, is something separate and potentially unrelated from the content labeled with the Veteran's name, rather than identifying the claimant information as a section within the bigger collection of information labeled with the Veteran's name.

Likewise, when the request is still pending, the heading structure is:
- H1: POA request
   - H2: [Claimant name]
   - H2: Claimant information
   - H2: Authorization information
         - H4: Do you accept or decline this POA request?

This improperly skips a heading level, and implies that the accept/decline question is a part of the authorization information.

<details><summary>Link, screenshot or steps to recreate</summary>

![H1 heading that reads "POA request" followed by an H2 heading that reads "Barrett, Chad."](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/99796-Brian-4.png?raw=true)

</details>

## Recommended action

I would recommend this instead:
- H1: POA request - [Claimant name]
   - H2: Claimant information
   - H2: Authorization information

I think that more faithfully represents the structure of the content. Note that you can style different parts of a heading however you'd like, so no visual changes are needed for how the Veteran's name is displayed so long as it's grouped in with the H1.

For pending requests, I would recommend:
- H1: POA request - [Claimant name]
   - H2: Claimant information
   - H2: Authorization information
   - H2: Do you accept or decline this POA request?

Again, totally okay for the "Do you accept" question to be styled the way it's styled now, so long as it has the right semantics under the hood.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by seipeljl** [comment]
@candilemoine @eheintzelman I've reviewed the ticket from a design and content standpoint and agree that we should make the updates suggested in the **Recommended action** section above, to leave the headers styled as they already are, but update the heading semantics.

Can we transition this into a frontend ticket? 

cc: @jquispe-oddball

**Comment by candilemoine** [comment]
I'll re-assign and move it back to `Ready`

**Comment by jquispe-oddball** [comment]
@candilemoine this is on staging but looks like it's broken right now, moving to qa

**Comment by jquispe-oddball** [comment]
@candilemoine this is fixed on staging, see screenshot. It is an h2, but matching the heading styling from figma
cc: @seipeljl 

![Image](https://github.com/user-attachments/assets/e9cfbf38-d999-425d-920e-c3fd4fd75ce3)

**Comment by candilemoine** [comment]
Thank you! LGTM 👍
